### PR TITLE
Dgo wg c1

### PR DESCRIPTION
**Pull request by dannago**
White glove & references for C

### CODE REVIEW

File: sp800-63c/sec4_fal.md [hunk 1]
```diff
@@ -25,13 +25,13 @@ This table presents different requirements for each FAL. Each successive level s
...
 
-Regardless of what is requested or required by the protocol, the FAL in use is easily detected by the RP by observing the nature of the assertion as it is presented as part of the federation protocol. Therefore, the RP is responsible for determining which FALs it is willing to accept for a given authentication transaction and ensuring that the transaction meets the requirements of that FAL.
+Regardless of what the protocol requests or requires, the RP easily detects the FAL in use by observing the nature of the assertion as it is presented as part of the federation protocol. Therefore, the RP is responsible for determining which FALs it is willing to accept for a given authentication transaction and ensuring that the transaction meets that FAL's requirements.
```
Comment (jricher) [inline sp800-63c/sec4_fal.md]: The protocol is not doing the requesting here, but I can see how the original phrasing could be confusing. Suggest rewording as:

> Regardless of what the RP requests or what the protocol requires, the RP can easily detect the FAL in use by observing...

**Review by jricher — CHANGES_REQUESTED** [review]

File: sp800-63c/sec4_fal.md (hunk 1 above)
Comment (dannago) [reply sp800-63c/sec4_fal.md]: Updated

File: sp800-63c/references.md [hunk 2]
```diff
@@ -5,30 +5,43 @@
 
 *This section is informative.*
 
+### 1. General References
```
Comment (jimfenton) [inline sp800-63c/references.md]: Should be section 12.1 (no trailing dot)

File: sp800-63c/references.md [hunk 3]
```diff
@@ -5,30 +5,43 @@
...
 <a name="M-03-22"></a>[M-03-22] OMB Memorandum M-03-22, *OMB Guidance for Implementing the Privacy Provisions of the E-Government Act of 2002*, September 26, 2003, available at: <https://georgewbush-whitehouse.archives.gov/omb/memoranda/m03-22.html>.
 
-<a name="RFC7591"></a>[RFC 7591] IETF, *OAuth 2.0 Dynamic Client Registration Protocol*, available at: <https://tools.ietf.org/html/rfc7591/>.
+<a name="nistir8112"></a>[NISTIR 8112] NIST Internal Report 8112 (Draft), *Attribute Metadata*, available at: <https://pages.nist.gov/NISTIR-8112/NISTIR-8112.html>.
 
-<a name="RFC7636"></a>[RFC 7636] IETF, *Proof Key For Code Exchange*, available at: <https://tools.ietf.org/html/rfc7636/>.
+<a name="Section508"></a>[Section 508] Section 508 Law and Related Laws and Policies (January 30, 2017), available at: <https://www.section508.gov/content/learn/laws-and-policies>.
 
-<a name="nistir8112"></a>[NISTIR 8112] NIST Internal Report 8112, *Attribute Metadata*, available at: <https://pages.nist.gov/NISTIR-8112/NISTIR-8112.html>.
+### 2. Standards
```
Comment (jimfenton) [inline sp800-63c/references.md]: 12.2

File: sp800-63c/references.md [hunk 4]
```diff
@@ -5,30 +5,43 @@
...
 
-<a name="SAML"></a>[SAML] OASIS, *Security Assertion Markup Language (SAML) V2.0 Technical Overview*, available at: <http://docs.oasis-open.org/security/saml/Post2.0/sstc-saml-tech-overview-2.0.html>.
+<a name="RFC7636"></a>[RFC 7636] IETF, *Proof Key For Code Exchange*, RFC 7636, DOI 10.17487/RFC7636, September 2015, <https://doi.org/10.17487/RFC7636>.
 
-<a name="OIDC"><a>[OIDC] OpenID Foundation, *OpenID Connect Core 1.0 incorporating errata set 1*, available at: <http://openid.net/specs/openid-connect-core-1_0.html>.
+<a name="SAML"></a>[SAML] OASIS, *Security Assertion Markup Language (SAML) V2.0 Technical Overview*, March 2008, available at: <http://docs.oasis-open.org/security/saml/Post2.0/sstc-saml-tech-overview-2.0.html>.
 
-<a name="FIPS140"></a>[FIPS 140] Federal Information Processing Standard Publication 140-2, *Security Requirements for Cryptographic Modules*, May 25, 2001 (with Change Notices through December 3, 2002). Available at:<http://csrc.nist.gov/publications/fips/>.
+### 3. NIST Special publications
```
Comment (jimfenton) [inline sp800-63c/references.md]: 12.3
publications --> Publications

File: sp800-63c/references.md [hunk 5]
```diff
@@ -5,30 +5,43 @@
...
-<a name="FEDRAMP"></a>[FEDRAMP] General Services Administration, *Federal Risk and Authorization Management Program*, available at: <https://www.fedramp.gov/>.
+<a name="SP800-53"></a>[SP 800-53] NIST Special Publication 800-53 Revision 4, *Recommended Security and Privacy Controls for Federal Information Systems and Organizations*, April 2013 (updated January 22, 2015), <http://dx.doi.org/10.6028/NIST.SP.800-53r4>.
+
+<a name="SP800-63-3"></a>[SP 800-63-3] NIST Special Publication 800-63-3, *Digital Identity Guidelines*, June 2017, <https://doi.org/10.6028/NIST.SP.800-63-3>.
+
+<a name="SP800-63A"></a>[SP 800-63A] NIST Special Publication 800-63A, *Digital Identity Guidelines: Enrollment and Identity Proofing Requirements*, June 2017, <https://doi.org/10.6028/NIST.SP.800-63a>.
+
+<a name="SP800-63B"></a>[SP 800-63B] NIST Special Publication 800-63B, *Digital Identity Guidelines: Authentication and Lifecycle Management*, June 2017, <https://doi.org/10.6028/NIST.SP.800-63b>.
+
+### 4. Federal Information Processing Standards
```
Comment (jimfenton) [inline sp800-63c/references.md]: 12.4

File: sp800-63c/sec10_usability.md [hunk 6]
```diff
@@ -40,13 +40,13 @@ Federated identity systems should:
...
+* Make trust relationships explicit to users to facilitate informed trust decisions. Trust relationships are often dynamic and context dependent. Users may be more likely to trust some IdPs and RPs with certain attributes or transactions more than others. For example, users may be more hesitant to use federated identity systems on websites that contain valuable personal information (such as financial or health). Depending on the perceived sensitivity of users' personal data, users may be less comfortable with social network providers as IdPs since people are often concerned with the broadcasting nature of social networking implementations.
 
-* Follow the usability considerations specified in 800-63A, Section 9 for any user-facing information.
+* Follow the usability considerations specified in SP 800-63A, Section 9 for any user-facing information.
```
Comment (jimfenton) [inline sp800-63c/sec10_usability.md]: Maybe add a link?

**Review by jimfenton — COMMENTED** [review]
Comments mostly relate to the references section. Also wonder why IETF references list the DOI in the text as well as in the URL, and others don't.

**Review by jimfenton — CHANGES_REQUESTED** [review]
From previously submitted review

File: sp800-63c/sec10_usability.md (hunk 6 above)
Comment (dannago) [reply sp800-63c/sec10_usability.md]: unsure how to link between volumes

File: sp800-63c/references.md [hunk 7]
```diff
@@ -5,30 +5,43 @@
...
 
-<a name="ISO9241-11"></a>[ISO 9241-11] International Organization for Standardization, *Ergonomic requirements for office work with visual display terminals (VDTs) — Part 11: Guidance on Usability*, ISO 9241-11:1998, ISO: Geneva, Switzerland, 1998.
+<a name="ISO9241"></a>[ISO 9241-11] International Standards Organization, ISO/IEC 9241-11 *Ergonomic requirements for office work with visual display terminals (VDTs) — Part 11: Guidance on usability*, 1998, available at: <https://www.iso.org/standard/16883.html>.
 
-<a name="Section508"></a>[Section 508] Section 508 Law and Related Laws and Policies (January 30, 2017), available at: <https://www.section508.gov/content/learn/laws-and-policies>.
+<a name="OpenIDConnectCore"></a>[OpenID Connect Core 1.0] Sakimura, N., Bradley, B., Jones, M., de Medeiros, B., and C. Mortimore, *OpenID Connect Core 1.0 incorporating errata set 1*, November, 2014. Available at: <https://openid.net/specs/openid-connect-core-1_0.html>.
```
Comment (jimfenton) [inline sp800-63c/references.md]: If changing the reference callout and anchor, the reference(s) to it also need to change (section 11 line 83)

File: sp800-63c/references.md [hunk 8]
```diff
@@ -5,30 +5,43 @@
...
 
-<a name="bcp195"></a>[BCP 195] Sheffer, Y., Holz, R., and P. Saint-Andre, *Recommendations for Secure Use of Transport Layer Security (TLS) and Datagram Transport Layer Security (DTLS)*, BCP 195, RFC 7525, May 2015, <http://www.rfc-editor.org/info/bcp195>.
+<a name="RFC4120"></a>[RFC 4120] IETF, *The Kerberos Network Authentication Service (V5)*,RFC 4120, DOI 10.17487/RFC4120, July 2005, <https://doi.org/10.17487/RFC4120>.
```
Comment (jimfenton) [inline sp800-63c/references.md]: space between comma and RFC

File: sp800-63c/sec10_usability.md (hunk 6 above)
Comment (jimfenton) [reply sp800-63c/sec10_usability.md]: It should look like:
> \[SP 800-63A, Section 9](sp800-63a.html#sec9)

File: sp800-63c/sec6_assertions.md [hunk 9]
```diff
@@ -14,37 +14,37 @@ All assertions SHALL include the following assertion metadata:
 1. Subject: An identifier for the party that the assertion is about (i.e., the subscriber).
 2. Issuer: An identifier for the IdP that issued the assertion.
 3. Audience: An identifier for the party intended to consume the assertion (i.e., the RP).
-4. Issuance: A timestamp indicating when the assertion was issued by the IdP.
+4. Issuance: A timestamp indicating when the IdP issued assertion.
```
Comment (jimfenton) [inline sp800-63c/sec6_assertions.md]: issued assertion --> issued the assertion

**Review by jimfenton — CHANGES_REQUESTED** [review]
A few more, mostly references-related

File: sp800-63c/references.md [hunk 10]
```diff
@@ -5,30 +5,43 @@
...
+<a name="RFC7636"></a>[RFC 7636] IETF, *Proof Key For Code Exchange*, RFC 7636, DOI 10.17487/RFC7636, September 2015, <https://doi.org/10.17487/RFC7636>.
 
-<a name="OIDC"><a>[OIDC] OpenID Foundation, *OpenID Connect Core 1.0 incorporating errata set 1*, available at: <http://openid.net/specs/openid-connect-core-1_0.html>.
+<a name="SAML"></a>[SAML] OASIS, *Security Assertion Markup Language (SAML) V2.0 Technical Overview*, March 2008, available at: <http://docs.oasis-open.org/security/saml/Post2.0/sstc-saml-tech-overview-2.0.html>.
 
-<a name="FIPS140"></a>[FIPS 140] Federal Information Processing Standard Publication 140-2, *Security Requirements for Cryptographic Modules*, May 25, 2001 (with Change Notices through December 3, 2002). Available at:<http://csrc.nist.gov/publications/fips/>.
+### 12.3 NIST Special Publications
+IST 800 Series Special Publications are available at: <http://csrc.nist.gov/publications/nistpubs/index.html>. The following publications may be of particular interest to those implementing systems of applications requiring e-authentication.
```
Comment (jimfenton) [inline sp800-63c/references.md]: IST --> NIST

File: sp800-63c/references.md (hunk 7 above)
Comment (dannago) [reply sp800-63c/references.md]: not sure I understand

Comment (jricher) [reply sp800-63c/references.md]: Previous anchor was "OIDC" and new anchor is "OpenIDConnectCore" but the references to that anchor weren't updated to use the new anchor.

**Review by jricher — APPROVED** [review]

File: sp800-63c/references.md [hunk 11]
```diff
@@ -19,7 +19,7 @@
 
 <a name="ISO9241"></a>[ISO 9241-11] International Standards Organization, ISO/IEC 9241-11 *Ergonomic requirements for office work with visual display terminals (VDTs) — Part 11: Guidance on usability*, 1998, available at: <https://www.iso.org/standard/16883.html>.
 
-<a name="OpenIDConnectCore"></a>[OpenID Connect Core 1.0] Sakimura, N., Bradley, B., Jones, M., de Medeiros, B., and C. Mortimore, *OpenID Connect Core 1.0 incorporating errata set 1*, November, 2014. Available at: <https://openid.net/specs/openid-connect-core-1_0.html>.
+<a name="OIDC"></a>[OpenID Connect Core 1.0] Sakimura, N., Bradley, B., Jones, M., de Medeiros, B., and C. Mortimore, *OpenID Connect Core 1.0 incorporating errata set 1*, November, 2014. Available at: <https://openid.net/specs/openid-connect-core-1_0.html>.
```
Comment (jimfenton) [inline sp800-63c/references.md]: The link to the anchor matches now, but the callout still doesn't. The callout to the reference in Section 11 line 83 is to [OIDC] and the reference here is to [OpenID Connect Core 1.0]. They should match.

**Review by jimfenton — APPROVED** [review]
